aggregation-worker/scripts/docker-build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 set -ex
-docker build -t javaduck .
+sudo docker build -t javaduck .
```

Comment:
Why not add jenkins to the docker group?

Reply:
Yes, my understanding is that dockerd requires root but everything else should be run as a regular user. sudo should be avoided, because it's nearly always a case of something like file system permissions rather than actually requiring root access. Unless it is something like a service doing root only things in the kernel, or trying to bind on a restricted port, then non-root should be used everywhere.

https://docs.docker.com/engine/security/security/